tests/integration/targets/virtualdirectory/tasks/main.yml

```
@@ -0,0 +1,80 @@
+- name: Run on Server 2016 and higher
+  block:
```

The name isn't needed here, seems to be a leftover from the older tests.

```suggestion
- block:
```

---

removed